nixos/modules/tasks/filesystems.nix

```
@@ -280,7 +280,8 @@ in
     environment.etc.fstab.text =
       let
         fsToSkipCheck = [ "none" "bindfs" "btrfs" "zfs" "tmpfs" "nfs" "vboxsf" "glusterfs" "apfs" ];
```

While you're on it It may be interesting to add Parallels and VmWare FS: 
```suggestion
        fsToSkipCheck = [ "none" "bindfs" "btrfs" "zfs" "tmpfs" "nfs" "vboxsf" "glusterfs" "apfs" "prl_fs" "vmhgfs" ];
```